internal/impl/devbox_test.go

```diff
@@ -43,10 +43,9 @@ func testShellPlan(t *testing.T, testPath string) {
 		assert := assert.New(t)
 
 		_, err := Open(&devopt.Opts{
-			Dir:          baseDir,
-			Writer:       os.Stdout,
-			ShowWarnings: true,
```

Comment:
We don't need `ignoreWarnings: false` for tests?

Reply:
false is default